circuit_puzzles/programs/announcer_configure.clsp

```diff
@@ -0,0 +1,88 @@
+; governance backdoor
+; we either activate this announcer or deactivate, can happen at any time via CRT vote
+(mod ((MOD_HASH STATUTES_STRUCT LAUNCHER_ID INNER_PUZZLE_HASH APPROVED DEPOSIT DELAY ATOM_VALUE COOLDOWN_START TIMESTAMP_EXPIRES)
+      ; solution
+      inner_puzzle_hash new_puzzle_hash deposit input_conditions
+      (current_timestamp statutes_inner_puzzle_hash input_toggle_activation deactivation_cooldown_interval delay new_price melt)
+     )
+  (include condition_codes.clib)
+  (include curry.clib)
+  (include *standard-cl-23*)
+  (include utils.clib)
+  (include statutes_utils.clib)
+  (include announcer.clib)
+
+  (assign
+    statutes_puzzle_hash (calculate-statutes-puzzle-hash STATUTES_STRUCT statutes_inner_puzzle_hash)
+    (toggle_activation . cooldown_start) (
+      if
+        (any (= input_toggle_activation 0) (= input_toggle_activation 1))
+        (if
+          (= input_toggle_activation 0)
+          (if
+            (all
+              APPROVED
+              (> COOLDOWN_START 0)
+              (> (- current_timestamp COOLDOWN_START) deactivation_cooldown_interval)
+            )
+            ; deactivating announcer
+            (c 0 0)
+            (if (= COOLDOWN_START 0)
+              ; starting the cooldown
+              (c 1 (+ deactivation_cooldown_interval current_timestamp))
```

Comment:
shouldn't starting cooldown set COOLDOWN to current_timestamp?

Reply:
yeah, that's a bug